src/api/data/cache/charthelper/chart_package_helper.go

```diff
@@ -134,6 +136,19 @@ var ensureChartDataDir = func(chart *models.ChartPackage) error {
 	return nil
 }
 
+var cleanChartDataDir = func(chart *models.ChartPackage) error {
```

Comment:
This is something that we're doing for functions that need to be overridden in tests, although I'm not actually overriding it in this case. I'm not sure if it makes sense to only use a variable where we intend to test it that way. What do you think?